scania-digital-design-system/tegel
# chore: fix Windows test script

**Describe pull-request**  
Refactored the unit testing script to be able to run on Windows OS.

**Solving issue**  
Fixes: [CDEP-2969](https://tegel.atlassian.net/jira/software/projects/CDEP/boards/1?selectedIssue=CDEP-2969)

**How to test**  
1. Check out branch
2. Cd into packages/core
3. Run "npm run test"

[CDEP-2969]: https://tegel.atlassian.net/browse/CDEP-2969?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
